### PR TITLE
Specify GET Method to Load templates

### PR DESCRIPTION
**Pull request by santiph**
RemoteStorage is using `POST` to load templates
Issue: https://github.com/artf/grapesjs/issues/436

### CODE REVIEW

File: src/storage_manager/model/RemoteStorage.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ module.exports = require('backbone').Model.extend({
   },
 
   load(keys, clb) {
-    this.request(this.get('urlLoad'), {body: {keys}}, clb);
+    this.request(this.get('urlLoad'), {body: {keys}, method: 'get'}, clb);
```
Comment (santiph) [inline src/storage_manager/model/RemoteStorage.js]: Body on GET?? @.@

**Comment by artf** [comment]
One of the ideas of `urlLoad` is too get only requested resources by using `keys` (eg. `["html", "css", ...]`). Unfortunately, the fetch specs don't allow an 'easy' way to add query parameters (https://github.com/github/fetch/issues/256) so this is why I left it as it is 😬
So before merging, I'd say we need to transform that array in a query string (eg. `keys[]=val&key[]=val2`)

**Comment by santiph** [comment]
What about a comma separated list of keys? `/templates/42?keys=assets,css,styles,html,components`

**Comment by artf** [comment]
What do you think about removing `keys` completely? If anybody need to fetch one resource or more all you have to do is to update `urlLoad` with your `keys` and logic.

I don't even see the point of adding `bodilessMethods` check, grapesjs's fetch polyfill was made to fit its own needs, if someone will use it for other scenarios it's not my responsability to check their requests, moreover you're applying the same logic twice (in `RemoteStorage.js` and `fetch.js`) :)

So, if you just remove the `keys`, from the `load` method, I'm gonna merge it

**Comment by artf** [comment]
Thanks @santiph 👍

**Comment by santiph** [comment]
Thanks to you, @artf  ;) 
Would you mind to publish a new version of grapesjs into npm?

**Comment by artf** [comment]
published

**Pull request merged**